compose/blockfrost-ryo/local-network/blockfrost.yaml

```diff
@@ -0,0 +1,9 @@
+server:
+  listenAddress: 0.0.0.0
+  port: 3000
+  prometheusMetrics: false
+  debug: true
+network: "custom"
+tokenRegistryUrl: "https://metadata.cardano-testnet.iohkdev.io"
```

Comment:
If this is a required configuration for blockfrost-ryo ok, otherwise it's useless as tokens in our local network have no metadata on public servers.

Reply:
I was hoping to replace it with a local network alternative, but seems not possible.  I should remove it with the next push.

Reply:
@iccicci This turned out to be mandatory. pm2 gave error inside docker if absent.